scripts/ubuntu/hello-world-deploy.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+if ( dpkg -l | grep npm ) && ( dpkg -l | grep node )
```

Comment:
use these checks to know whether you should install or not instead of `which X`. And use `dpkg -s X` maybe as it may be more specific to things "being installed"?